endpoints/put_test.go

```diff
@@ -482,22 +482,27 @@ func TestRequestReadError(t *testing.T) {
 func TestTooManyPutElements(t *testing.T) {
 	// Test case: request with more than elements than put handler's max number of values
 	putElements := []string{
-		"{\"type\":\"json\",\"value\":true}",
-		"{\"type\":\"xml\",\"value\":\"plain text\"}",
-		"{\"type\":\"xml\",\"value\":\"2\"}",
+		`{"type":"json","value":true}`,
+		`{"type":"xml","value":"plain text"}`,
+		`{"type":"xml","value":"2"}`,
 	}
-	reqBody := fmt.Sprintf("{\"puts\":[%s, %s, %s]}", putElements[0], putElements[1], putElements[2])
+	reqBody := fmt.Sprintf(`{"puts":[%s, %s, %s]}`, putElements[0], putElements[1], putElements[2])
 
 	//Set up server with capacity to handle less than putElements.size()
-	backend := backends.NewMemoryBackend()
+	backend := &mockBackend{}
+	backend.On("Put", mock.Anything, mock.Anything, mock.Anything, mock.Anything).Return(nil)
+
 	router := httprouter.New()
 	m := metricstest.CreateMockMetrics()
 	router.POST("/cache", NewPutHandler(backend, m, len(putElements)-1, true))
 
-	_, httpTestRecorder := doMockPut(t, router, reqBody)
-	assert.Equalf(t, http.StatusBadRequest, httpTestRecorder.Code, "doMockPut should have failed when trying to store %d elements because capacity is %d ", len(putElements), len(putElements)-1)
+	putResponse := doPut(t, router, reqBody)
+
+	assert.Equalf(t, http.StatusBadRequest, putResponse.Code, "doMockPut should have failed when trying to store %d elements because capacity is %d ", len(putElements), len(putElements)-1)
```

Comment:
Nitpick: the string at the end refers to `doMockPut` but it should now be `doPut`

Reply:
Corrected